docs/environment

```diff
@@ -22,6 +22,12 @@
 # XKB_DEFAULT_OPTIONS=grp:alt_shift_toggle
 # XKB_DEFAULT_OPTIONS=grp:shift_caps_toggle
 
+## GTK4 started to require input methods like fcitx5 or ibus to handle
```

Comment:
Suggest saying from version 4.20.0.
Consider explaining what compose keys are, for example typing “à” with “`” + “a”.
It may also be worth pointing out what "input methods" are - e.g. `Fcitx5` and `IBus`
See: https://labwc.github.io/integration.html#gtk

Reply:
Good idea. Makes sense to just document in one place.

Reply:
added the link